core/src/main/java/bio/terra/pearl/core/service/workflow/RegistrationService.java

```diff
@@ -125,6 +131,32 @@ private RegistrationResult register(String portalShortcode, EnvironmentName envi
         return new RegistrationResult(user, ppUser);
     }
 
+    public RegistrationResult registerGovernedUser(String portalShortcode, ParticipantUser proxy) {
```

Comment:
Before this method is called, the parent/proxy user has been registered and exists in participant user. In time of enrollment, if the server sees that the frontend wants to enroll someone else as the enrollee (for example the child of the proxy) this method registers the child (inserts a record in the participant_user) before enrolling them